### ISSUE TITLE
AppStore Testflight Release

### ISSUE THREAD

**Issue by theneedyguy** [supergeorg/Grocy-SwiftUI#1]
Is a release on the AppStore planned? An initial TestFlight release would be nice to help with beta testing.

**Comment by supergeorg** [comment]
An AppStore release is of course planned. I will look into the registration in the developer program.
For now, I will add unsigned test builds for macOS and iOS in the release section. These have to be sideloaded manually.

**Comment by supergeorg** [comment]
Sorry for the long delay, I took some time to setup my payment method and Apple Developer.
But now the app is in review by Apple.

Also there is a signed release in the releases section (for macOS only), which will be the only way for macOS before App Store release, since TestFlight is not available on macOS (don't know why).
I will post here when it is approved.

**Comment by supergeorg** [comment]
The App is now released on App Store for iOS, the macOS version is pending. The link is in the readme file.

**Comment by Floflobel** [comment]
Thank you for this publication on the apple store. I had tried to install the application on my iPhone jailbreak and I had not succeeded because of an error.
Is it possible to allow the installation of all versions under iOS 14? It is necessary to have the version 14.5 currently.

**Comment by supergeorg** [comment]
The minimum supported version is 14.5, if I changed some configuration I could go down to 14.0 (nobody uses this version anymore).
But versions lower than 14 (e.g. 13, 12) are not supported, since the App is developed as Multiplatform and with the new "@main", requiring iOS 14.

**Comment by Floflobel** [comment]
The best for me would be to lower the version to 14.3. 
There are still people on the older versions like 14.0 but I understand that there are not many people left.

Do you want me to open a new ticket to lower the minimum version to 14.3?

**Comment by supergeorg** [comment]
Are there any particular reason to using iOS 14.3? I have heard that there are serious security holes in there and the update to 14.5/14.6 seems flawless.
I have set the version down to 14.3, the next update (may take some time, will release only when I have added/fixed some things) should support it. This shouldn't be a problem since it is just a version string, but is not a priority for aforementioned reasons.
